SonarQube.Bootstrapper/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ERROR_FailedToUpdateRunnerBinaries" xml:space="preserve">
     <value>Failed to update the SonarQube MSBuild Runner binaries. Check the server url, verify that the C# plugin is correctly installed on the SonarQube server and that the SonarQube server has been restarted.</value>
   </data>
+  <data name="ERROR_PostProcessExeNotFound" xml:space="preserve">
+    <value>Post-processor executable not found '{0}'. Please check that pre-processing was executed in the current directory.</value>
```

Comment:
@HSAR "in the same directory" might be better; it's slightly more generic and will make more sense if the user isn't building from the command line.